src/radical/pilot/task_manager.py

```diff
@@ -640,6 +646,26 @@ def control_cb(self, topic, msg):
             self._rpc_queue.put(arg)
 
 
+        elif cmd == 'service_up':
+
+            self._log.debug('=== service up: %s', arg)
+            self._service_update(arg)
+
+
+    # --------------------------------------------------------------------------
+    #
+    def _service_update(self, arg):
+
+        uid  = arg['uid']
+        task = self._tasks.get(uid)
+
+        if not task:
+            return
+
+        task._set_info(arg['info'] or '')  # `None` would indicate an error
```

Comment:
related to the comment above, and it will come from agent_0:L438

Reply:
ack